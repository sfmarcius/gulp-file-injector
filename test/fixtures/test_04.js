//! $file(./test_03.js)
function test_04() {
    return "test_04";
}
