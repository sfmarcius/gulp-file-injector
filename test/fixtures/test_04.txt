$file(./test_03.txt)
test_04() {
    return "test_04";
}
