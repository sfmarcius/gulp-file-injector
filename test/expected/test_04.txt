test_01() {
    return "test_01";
}
test_02() {
    return "test_02";
}
test_03() {
    return "test_03";
}
test_04() {
    return "test_04";
}
